src/middlewares/errorMiddleware.js

```diff
@@ -0,0 +1,9 @@
+function errorMiddleware(err, req, res, next) {
+  if (err) {
+    res.status(503).json({ message: 'Server error.' });
```

Comment:
All errors you can move in global constants

Reply:
When I left this I knew that you would not come through)))